pymedphys/tests/dicom/test_anonymise.py

```diff
@@ -31,10 +32,9 @@
 from pymedphys._dicom.utilities import remove_file
 from pymedphys.dicom import anonymise as anonymise_dataset
 
-HERE = dirname(abspath(__file__))
-DATA_DIR = pjoin(HERE, "data", "anonymise")
-TEST_FILEPATH = pjoin(DATA_DIR, "RP.almost_anonymised.dcm")
-
+DATA_PATHS = download.zip_data_paths("rtplan-anonymisation.zip")
```

Comment:
By placing this as a constant at the top of the file this will make it so that this data is downloaded even it is not used.

Could this file path logic either be placed at the top of the function using it, or if it is used by multiple functions have it be placed either within a pytest.fixture (https://docs.pytest.org/en/stable/fixture.html), or a function inside a functools.lrucache decorator?